### ISSUE TITLE
A11y_Azure Data Studio extension 'Azure SQL Migration'_Create new Azure Database migration service_Link Purpose: Narrator/NVDA is not announcing Create new link descriptively.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/azuredatastudio#19861]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### Github Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #NonBenchmark; #SH-AzureSQLMigration-Win32-Jun2022; #DesktopApp; #Win32; #SQL Migration Control Plane; #WCAG2.4.4; #Link Purpose;

### Environment and OS details:
Azure Data Studio Version 1.37.0(User Setup)
Windows Version 21H2 (OS Build 22000.675)

### Note:
This issue is observed for Create new link for resource group present Azure Database migration service dialog

### Repro Steps:

1. Turn on Narrator/NVDA
2. Install Azure data studio app and install Azure SQL migration extension
3. Connect to SQL server, tab till respective server, right click and select manage
4. Tab till Azure SQL Migration in sitemap and press enter key
5. Tab till migrate to azure sql and press enter
6. Navigate to databases grid in "Databases for assessment", select few databases and click next.
7. Fill all the required fields and navigate to Azure Database migration service(step 6 ) and tab till create new link
8. Verify whether screen reader is announcing create new link descriptively or not

### Actual Result:
Narrator/NVDA is not announcing Create new link descriptively. Screen reader is just announcing as "Create new" but it is not announcing what can be newly created

### Expected Result:
Narrator/NVDA should announce Create new link descriptively. Screen reader should clearly announce what can be newly created i.e. Create new Azure Database migration service link or Create new Resource group link

### Attachment:
![NVDA_create new](https://user-images.githubusercontent.com/83570638/176377633-da4c06c8-f894-4e23-9184-f239eeddbcb1.png)
![Narrator_create new_](https://user-images.githubusercontent.com/83570638/176377728-ce55359e-43b5-42e5-b5e0-90fd2ed620f2.gif)

**Comment by aasimkhan30** [comment]
@brian-harris, can you please add aria-label to these links with a better description?

**Comment by VijayaBharathyA** [comment]
#Closed;
Verified the issue in below Environment:
Issue is Fixed
Version: 1.41.1 (user setup)
![Screenshot (28)](https://user-images.githubusercontent.com/100189407/218050299-4178c8ee-52a6-4f7e-96fe-6b1b9bd5f7e7.png)
![Screenshot (27)](https://user-images.githubusercontent.com/100189407/218050307-97d71eeb-34a4-4566-bfbe-daee2b0c622f.png)
